src/harness/testcases/WINNF_FT_S_WDB_testcase.py

```diff
@@ -298,8 +298,8 @@ def test_WINNF_FT_S_WDB_2(self, config_filename):
     files = {('/' + cpi['publicKeyIdentifierFile']):
              cpi['publicKeyIdentifierFile']
              for cpi in config['cpiDatabaseConfig']['cpis']}
-    files['/' + config['cpiDatabaseConfig']['indexUrl']] = config[
-        'cpiDatabaseConfig']['indexUrl']
+    files['/' + config['cpiDatabaseConfig']['indexPath']] = config[
```

Comment:
The first should be 'indexUrl' and the second should be 'indexPath', as described in database.py.

Reply:
yes, should be good now. please take a look.